tutordiscovery/plugin.py

```diff
@@ -34,13 +35,19 @@
         "EXTRA_PIP_REQUIREMENTS": [],
         "REPOSITORY": "https://github.com/openedx/course-discovery.git",
         "REPOSITORY_VERSION": "{{ OPENEDX_COMMON_VERSION }}",
+        "DEV_BASE_URL": "http://{{DISCOVERY_HOST}}:8381",
+        "DEV_MEDIA_URL": "http://{{DISCOVERY_HOST}}:8381/media/"
     },
     "unique": {
         "MYSQL_PASSWORD": "{{ 8|random_string }}",
         "SECRET_KEY": "{{ 20|random_string }}",
         "OAUTH2_SECRET": "{{ 8|random_string }}",
         "OAUTH2_SECRET_SSO": "{{ 8|random_string }}",
     },
+    "overrides": {
+        "DISCOVERY_BASE_URL": PROTOCOL + "://{{DISCOVERY_HOST}}",
+        "MEDIA_URL": PROTOCOL + "://{{DISCOVERY_HOST}}/media/",
+    }
```

Comment:
Let's not introduce new Tutor settings here. Instead, the templates should be like:

```
DISCOVERY_BASE_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ DISCOVERY_HOST }}:8381"
MEDIA_URL = DISCOVERY_BASE_URL + "/media"
```